test/test_core.py

```diff
@@ -6264,7 +6264,8 @@ def test_unicode_js_library(self):
         return 0;
       }
     ''')
-    self.emcc_args += ['--js-library', test_file('unicode_library.js')]
+    create_file('modularize_post.js', '(async function main(){await Module();})();')
```

Comment:
Do you need the wrapper function here it can you just do `await` at the top level?

Reply:
Top level `await` is supported in Node only in ES modules.